### PR TITLE
Fix LRO to have scopes passed to operations client

### PR DESCRIPTION
**Pull request by landrito**

### CODE REVIEW

File: src/test/java/com/google/api/codegen/testdata/nodejs_doc_main_library.baseline [hunk 1]
```diff
@@ -125,21 +125,22 @@ function LibraryServiceClient(gaxGrpc, grpcClients, opts) {
...
     servicePath: servicePath,
     port: port,
     sslCreds: sslCreds,
     clientConfig: clientConfig,
     appName: appName,
-    appVersion: appVersion
-  });
+    appVersion: appVersion,
+    scopes: ALL_SCOPES
+  }
```
Comment (jmuk) [inline src/test/java/com/google/api/codegen/testdata/nodejs_doc_main_library.baseline]: semicolon at the end of this line.

File: src/test/java/com/google/api/codegen/testdata/nodejs_doc_main_library.baseline [hunk 2]
```diff
@@ -125,21 +125,22 @@ function LibraryServiceClient(gaxGrpc, grpcClients, opts) {
...
+  var operationsClient = new gax.lro(operationsOpts).operationsClient(operationsOpts);
```
Comment (jmuk) [inline src/test/java/com/google/api/codegen/testdata/nodejs_doc_main_library.baseline]: `gax.lro` will accept a slightly different parameters. There's a little different things (like `keyFilename`) which are passed to `index.js`, not here -- that's a bit complicated.  You'll probably have to modify gax a little bit to allow having those information.

See also https://github.com/googleapis/gax-nodejs/blob/master/lib/grpc.js#L58 grpcClient passes `options` to auth.

File: src/test/java/com/google/api/codegen/testdata/nodejs_doc_main_library.baseline (hunk 1 above)
Comment (landrito) [inline src/test/java/com/google/api/codegen/testdata/nodejs_doc_main_library.baseline]: Done!

File: src/test/java/com/google/api/codegen/testdata/nodejs_doc_main_library.baseline (hunk 2 above)
Comment (landrito) [inline src/test/java/com/google/api/codegen/testdata/nodejs_doc_main_library.baseline]: Done! Thanks for clarifying this!

**Comment by landrito** [comment]
Thanks for the review!

**Pull request merged**